src/index.html

```diff
@@ -12,7 +12,7 @@
   </head>
   <body>
     <div class="todo-list">
-      <h3>Today's To Do.</h3>
+      <h3>Today's To Do</h3>
       <div class="add-task">
```

Comment:
Hello @Nejjari001, 
You have done great work with your project so far ✅

- Followed Javascript clean Code principles ✅
- No Linter errors ✅
- Followed Github workflow ✅
However, 
I suggest you use HTML semantics (ex: Sections, header, and main ) instead of using divs to make your code more readable and maintainable.
Happy Coding!